Automattic/jetpack
# Add evaluation form to Welcome Flow

## Proposed changes:
* Add Evaluation step with checkbox form
* Add Evaluation "Loader" step (with timeout, currently)
* Integrate Evaluation with Welcome Banner flow

![CleanShot 2024-06-19 at 15 56 06@2x](https://github.com/Automattic/jetpack/assets/8419292/44dc964b-c872-41e8-8939-c15e41fd032d)
![CleanShot 2024-06-19 at 15 56 18@2x](https://github.com/Automattic/jetpack/assets/8419292/e9443176-1915-447f-881f-f4bde215d617)



### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Create a JN site with Jetpack Beta `add/welcome-banner-survey` branch selected
* In My Jetpack, notice that you have a Welcome Banner presented
* Connect the site, and notice that the next step is Evaluation form
* After submitting the form - you should see "Evaluation processing" screen for around 3 seconds (mocked operation which will be addressed in the following PR) - after that the Welcome Banner will disappear
